cllc-public-app/Models.Extensions/SepDrinkTypes.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
```

Comment:
The two files below: **SepDrinkTypes.cs** and **SepEventDates.cs** seem to have the wrong content, like they would need to be swapped, right?